api.js

```diff
@@ -573,14 +575,80 @@ module.exports = done => {
         );
     }
 
-    server.get(
-        { path: '/openapi', name: 'openapi-docs-generation' },
-        tools.responseWrapper(async (req, res) => {
-            res.charSet('utf-8');
-
-            const routes = server.router.getRoutes();
-
-            tools.generateAPiDocs(routes);
+    server.pre(
+        restifyApiGenerate(server, {
+            openapiVersion: '3.0.0',
```

Comment:
Could you move this configuration structure into a separate file (json or js) where the config would be loaded from?